app/components/participant-creator/component.js

```diff
@@ -73,6 +73,12 @@ export default Ember.Component.extend({
             });
             if (!useAsPassword) {
                 password = makeId(10);
+                this.get('extra').pushObject({
+                    key: 'password',
+                    value: password
+                });
+                extra['password'] = password;
+                this.set('useAsPassword', 'password');
```

Comment:
`useAsPassword` is the name of the field whose value should be used as the password, not the actual password. Maybe `passwordField` is more clear?